test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/KieServerS2ISmokeIntegrationTest.java

```diff
@@ -0,0 +1,87 @@
+import org.kie.cloud.common.provider.KieServerClientProvider;
+import org.kie.cloud.integrationtests.AbstractCloudIntegrationTest;
+import org.kie.server.api.model.KieContainerResource;
+import org.kie.server.api.model.ReleaseId;
+import org.kie.server.api.model.definition.ProcessDefinition;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.client.QueryServicesClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class KieServerS2ISmokeIntegrationTest extends AbstractCloudIntegrationTest<GenericScenario> {
+
+    protected KieServicesClient kieServicesClient;
+    protected QueryServicesClient queryServicesClient;
+
+    private static final Logger logger = LoggerFactory.getLogger(KieServerS2ISmokeIntegrationTest.class);
+
+    private static final String KIE_CONTAINER_DEPLOYMENT = CONTAINER_ID + "=" + PROJECT_GROUP_ID + ":" + DEFINITION_PROJECT_NAME + ":" + DEFINITION_PROJECT_VERSION;
+
+    private static final String REPO_BRANCH = "master";
+    private static final String PROJECT_SOURCE_FOLDER = "/kjars-sources";
+    private static final String REPO_NAME = "KieServerS2ISmokeIntegrationTest";
+
+    @Override
+    protected GenericScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
+        gitProvider.createGitRepository(REPO_NAME, ClassLoader.class.getResource(PROJECT_SOURCE_FOLDER).getFile());
+
+        return deploymentScenarioFactory.getGenericScenarioBuilder()
+                .withKieServerS2I(KIE_CONTAINER_DEPLOYMENT, gitProvider.getRepositoryUrl(REPO_NAME), REPO_BRANCH, DEFINITION_PROJECT_NAME)
+                .build();
+    }
+
+    @Before
+    public void setUp() {
+        kieServicesClient = KieServerClientProvider.getKieServerClient(deploymentScenario.getKieServerDeployment());
+        queryServicesClient = KieServerClientProvider.getQueryClient(deploymentScenario.getKieServerDeployment());
+    }
+
+    @After
+    public void deleteRepo() {
+        gitProvider.deleteGitRepository(REPO_NAME);
+    }
+
+    @Test
+    public void testContainerAfterExecServerS2IStart() {
+        List<KieContainerResource> containers = kieServicesClient.listContainers().getResult().getContainers();
+        assertThat(containers).isNotNull().hasSize(1);
+
+        KieContainerResource container = containers.get(0);
+        assertThat(container).isNotNull();
+        assertThat(container.getContainerId()).isNotNull().isEqualTo(CONTAINER_ID);
+
+        ReleaseId containerReleaseId = container.getResolvedReleaseId();
+        assertThat(containerReleaseId).isNotNull();
+        assertThat(containerReleaseId.getGroupId()).isNotNull().isEqualTo(PROJECT_GROUP_ID);
+        assertThat(containerReleaseId.getArtifactId()).isNotNull().isEqualTo(DEFINITION_PROJECT_NAME);
+        assertThat(containerReleaseId.getVersion()).isNotNull().isEqualTo(DEFINITION_PROJECT_VERSION);
+
+        List<ProcessDefinition> processes = queryServicesClient.findProcesses(0, 20);
```

Comment:
I think it would be better to check functionality by completing process.
That would make the test independent on amount of processes in container.